Controller/Store/GetStoredetails.php

```diff
@@ -0,0 +1,195 @@
+\Magento\Framework\App\Config\ScopeConfigInterface $scopeConfig
+
+    )
+    {        
+      
+        $this->_storeManager = $storeManager;   
+    
+	$this->countryInformation = $countryInformation;
+	$this->scopeConfig = $scopeConfig;
+
+        parent::__construct($context);
+    }
+
+    /**
+     * Retrieve formatted store address from config
+     *
+     * @param Store $store
+     * @return string
+     */
+    public function getFormattedAddress(Store $store)
+    {
+        /** @var \Magento\Customer\Block\Address\Renderer\DefaultRenderer $renderer */
+        $renderer = $this->addressConfig->getFormatByCode('html')->getRenderer();
+        return $renderer->renderArray($this->getStoreInformationObject($store)->getData());
+    }
+    
+    /**
+     * Get store identifier
+     *
+     * @return  int
+     */
+    public function getStoreId()
+    {
+        return $this->_storeManager->getStore()->getId();
+    }
+    
+    /**
+     * Get website identifier
+     *
+     * @return string|int|null
+     */
+    public function getWebsiteId()
+    {
+        return $this->_storeManager->getStore()->getWebsiteId();
+    }
+    
+    /**
+     * Get Store code
+     *
+     * @return string
+     */
+    public function getStoreCode()
+    {
+        return $this->_storeManager->getStore()->getCode();
+    }
+    
+    /**
+     * Get Store name
+     *
+     * @return string
+     */
+    public function getStoreName()
+    {
+        return $this->_storeManager->getStore()->getName();
+    }
+    
+    /**
+     * Get current url for store
+     *
+     * @param bool|string $fromStore Include/Exclude from_store parameter from URL
+     * @return string     
+     */
+    public function getStoreUrl($fromStore = true)
+    {
+        return $this->_storeManager->getStore()->getCurrentUrl($fromStore);
+    }
+    
+    /**
+     * Check if store is active
+     *
+     * @return boolean
+     */
+    public function isStoreActive()
+    {
+        return $this->_storeManager->getStore()->isActive();
+    }
+
+   public function execute()
+    {
+         $resultJson = $this->resultFactory->create(ResultFactory::TYPE_JSON);     
+        try{
+        	$storeid = $this->getStoreId();
+		$websiteid = $this->getWebsiteId();
+		$storecode = $this->getStoreCode();
+		$storename = $this->getStoreName();
+		$storeurl = $this->getStoreUrl();
+
+//$conf = $this->_helper->getStoreConfig('app/code/RicardoMartins/PagSeguro/etc/frontend/di.xml');
+
+$objectManager = \Magento\Framework\App\ObjectManager::getInstance();
+
+$directory = $objectManager->get('\Magento\Framework\Filesystem\DirectoryList');
+
+$rootPath  =  $directory->getRoot();
+
+$compojson = file_get_contents($rootPath.'/app/code/RicardoMartins/PagSeguro/composer.json');
+
+$config_data= json_decode($compojson,true);
+ unset($config_data['autoload']);
+
+		$street_line1 = $this->scopeConfig->getValue('general/store_information/street_line1');
```

Comment:
we don't need this info. Exposing this could be seen by merchant's as unnecessary risk